frontend/src/views/CellGuide/components/CellGuideCard/components/MarkerGeneTables/index.tsx

```diff
@@ -191,234 +193,362 @@ export const MARKER_GENES_COMPUTATIONAL_TOOLTIP_TEST_ID =
+        })
+        .filter((publicationTitle, index) => {
+          return publicationTitle && publications[index];
+        });
+
+      publications = sortedPublicationsAndTitles.map((pub) => pub[0]);
+      publicationTitles = sortedPublicationsAndTitles.map((pub) => pub[1]);
+
+      const publicationLinks = (
+        <PublicationLinkWrapper>
+          {publicationTitles.map((publicationTitle, index) => {
+            if (publicationTitle && publications[index]) {
+              const referenceIndexLabel =
+                publicationTitlesToIndex.get(publicationTitle) + 1;
+              return (
+                <Tooltip
+                  key={`${publications[index]}-${index}-tooltip`}
+                  placement="top"
+                  width="default"
+                  arrow={false}
+                  title={
+                    <div>
+                      {publicationTitle.split("\n\n").at(0)}
+                      <br />
+                      <br />
+                      <i>{publicationTitle.split("\n\n").at(1)}</i>
+                    </div>
+                  }
+                  leaveDelay={0}
+                >
+                  <span key={`${publications[index]}-${index}-span`}>
+                    <Link
+                      key={`${publications[index]}-${index}`}
+                      label={`[${referenceIndexLabel}]`}
+                      url={`https://doi.org/${publications[index]}`}
+                    />
+                  </span>
+                </Tooltip>
+              );
+            }
+          })}
+        </PublicationLinkWrapper>
+      );
+
+      rows.push({
+        name,
+        symbolId: symbol,
+        symbol: (
+          <>
+            {symbol}{" "}
+            <ButtonIcon
+              aria-label={`display gene info for ${symbol}`}
+              sdsIcon="infoCircle"
+              sdsSize="small"
+              sdsType="secondary"
+              onClick={() => setGeneInfoGene(symbol.toUpperCase())}
+            />
+          </>
+        ),
+        references: publicationLinks,
+        numReferences: sortedPublicationsAndTitles.length,
+      });
+    }
+
+    // Sort rows by number of references
+    if (rows.length) {
+      rows.sort((a, b) => {
+        return b.numReferences - a.numReferences;
+      });
+    }
+
+    return {
+      selectedOrganFilter,
+      tableRows: rows,
+      uniqueOrgans: sortedOrgans,
+      uniqueOrganisms: sortedOrganisms,
+    };
+  }, [genes, selectedOrgan, setGeneInfoGene]);
+}
+
 const MarkerGeneTables = ({
   cellTypeId,
   cellTypeName,
   setGeneInfoGene,
 }: Props) => {
-  const [selectedOrganism, setSelectedOrganism] = useState("");
   // 0 is canonical marker genes, 1 is computational marker genes
   const [activeTable, setActiveTable] = useState(0);
-  const [selectedOrgan, setSelectedOrgan] = useState("");
-  const [page, setPage] = useState(1);
+  const [selectedOrganCanonical, setSelectedOrganCanonical] = useState("");
```

Comment:
added more state variables to track dropdown state for the canonical and computational marker gene tables separately to prevent weird edge cases where selected tissue in one table is an invalid choice for the other table